lib/pact/cli.rb

```diff
@@ -91,3 +106,12 @@ def process_argv(subcommand)
     end
   end
 end
+
+
+def execute_pact_rust_command command
```

Comment:
we don't need this change as part of this PR, it got pulled in the cherry pick

Reply:
removed